src/Patches/PlayerControllerPatch.cs

```diff
@@ -10,87 +12,116 @@ namespace OoLunar.LethalCompanyPatched.Patches
     [HarmonyPatch(typeof(PlayerControllerB)), LethalPatch]
     internal class PlayerControllerPatch
     {
-        private static readonly FieldInfo _playerCarryWeight = typeof(PlayerControllerB).GetField("carryWeight");
-        private static bool _tempCrouch = false;
+        private static readonly FieldInfo _playerCarryWeight = typeof(PlayerControllerB).GetField("carryWeight");            
+        private static readonly FieldInfo? _slipperinessConfigField = AccessTools.Field(typeof(LethalCompanyPatchedPlugin), nameof(LethalCompanyPatchedPlugin.Slipperiness));
+        private static readonly FieldInfo? _jumpDelayConfigField  = AccessTools.Field(typeof(LethalCompanyPatchedPlugin), nameof(LethalCompanyPatchedPlugin.JumpDelay));
+        private static readonly FieldInfo? _instantSprintConfigField = AccessTools.Field(typeof(LethalCompanyPatchedPlugin), nameof(LethalCompanyPatchedPlugin.InstantSprint));
+        private static readonly MethodInfo? _configEntryFloatValueMethod = AccessTools.Method(typeof(ConfigEntry<float>), "get_Value");
+        private static readonly MethodInfo? _configEntryBoolValueMethod  = AccessTools.Method(typeof(ConfigEntry<bool>), "get_Value");
```

Comment:
Auto-implemented properties usually have backing fields behind them. This means we can find that field and skip the getter.